### PR TITLE
Dianoga implementation

### PR DESCRIPTION
**Pull request by atul9953**
Dianoga implementation

### CODE REVIEW

File: up.ps1 [hunk 1]
```diff
@@ -43,6 +43,19 @@ if (-not $envCheck) {
 Write-Host "Keeping XM Cloud base image up to date" -ForegroundColor Green
 docker pull "$($sitecoreDockerRegistry)sitecore-xmcloud-cm:$($sitecoreVersion)"
 
+# *** DLL Copying Logic Starts Here ***
+$libDir = Join-Path $PSScriptRoot "src\lib"
+$binDir = Join-Path $PSScriptRoot "docker\deploy\platform\bin"
```
Comment (ramangupta44) [inline up.ps1:48]: @atul9953 servers dont have this path.

Comment (ramangupta44) [reply up.ps1:48]: Try $binDir = C:\inetpub\wwwroot\bin

File: up.ps1 [hunk 2]
```diff
@@ -43,6 +43,19 @@ if (-not $envCheck) {
...
+Write-Host "Copying DLL files from '$libDir' to '$binDir'" -ForegroundColor Cyan
+if (-not (Test-Path -Path $binDir)) {
```
Comment (ramangupta44) [inline up.ps1:50]: This is not required. We dont want to create a bin folder.

Comment (atul9953) [reply up.ps1]: changes done removed bin folder creation

**Review by ramangupta44 — APPROVED** [review]

**Pull request merged**